### ISSUE TITLE
v-b-tooltip width too narrow after update to 0.24.23

### ISSUE THREAD

**Issue by geozaur** [bootstrap-vue-next/bootstrap-vue-next#2255]
### Describe the bug

Hello! After upgrading to the latest version (0.24.23), all tooltips have a very narrow width. It worked on the previous version (0.24.22).

I tried commenting the line introduced in the latest version and it worked. But, of course, this means reverting https://github.com/bootstrap-vue-next/bootstrap-vue-next/pull/2234.
https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/packages/bootstrap-vue-next/src/utils/floatingUi.ts#L120

Here is a reproduction with the working behaviour on the previous version: https://stackblitz.com/edit/github-qpbouv-kd3mdm?file=package.json,src%2Fcomponents%2FComp.vue

Not working:
![image](https://github.com/user-attachments/assets/99556ff2-0708-4bd7-9050-2ccd99be394a)

Working:
![image](https://github.com/user-attachments/assets/2d8c05de-06f2-4b52-8378-9647879c686e)

### Reproduction

https://stackblitz.com/edit/github-qpbouv?file=src%2Fcomponents%2FComp.vue

### Used Package Manager

npm

### Validations

- [X] Have tested with the latest version. This is still alpha version and sometime things change rapidly.
- [X] Follow our [Code of Conduct](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CODE_OF_CONDUCT.md)
- [X] Read the [Contributing Guide](https://github.com/bootstrap-vue-next/bootstrap-vue-next/blob/main/CONTRIBUTING.md).
- [X] Check that there isn't already an issue that reports the same bug to avoid creating a duplicate.
- [X] Check that this is a concrete bug. For Q&A, please open a GitHub Discussion instead.
- [X] The provided reproduction is a [minimal reproducible](https://stackoverflow.com/help/minimal-reproducible-example) of the bug.

**Comment by xvaara** [comment]
Yes, the fix in that pr seems wrong to me. absolute creates a new stacking context that is inherited by the tooltip.
